code_review_guide.md

```
@@ -56,11 +106,42 @@
+
+* Consider separating feedback into three tiers: blocking issues, suggestions, and nit picks. These decrease in importance:
+
+    * **All blocking issues must be addressed or have a resolution/compromise worked out with the reviewer.**
+
+    * Suggestions and nit picks can be non-blocking, less important forms of feedback. It's up to the author to incorporate these forms of feedback, and it should not matter to the reviewer if they are addressed or not.
+
+        * A suggestion might be, for example, a different name for a variable or field that the reviewer thinks is clearer or more idiomatic. These can be blocking (for example, if the name is confusing or doesn't make sense), or non-blocking (if the existing name is fine, but could be improved). The reviewer should make their expectations clear.
+
+        * A nit pick might be, for example, fixing a typo or awkwardly-phrased comment, removing extraneous parentheses around a Boolean condition, or simplifying nested if conditionals with a Boolean `&&` operator.
+
+      * That said, be careful to not use their personal preferences as a reason to excessively nit pick.
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 8; Actual: 6

(MD005, list-indent)